packages/components/docs/migration/migrate-to-10.x.md

```diff
@@ -31,7 +31,7 @@ initial `v10` release.
 | Combobox           | No breaking changes                                                   |
 | Content Switcher   | [Migrate](../../src/components/content-switcher/migrate-to-10.x.md)   |
 | Copy Button        | [Migrate](../../src/components/copy-button/migrate-to-10.x.md)        |
-| Data Table V2      | [Migrate](../../src/components/data-table/migrate-to-10.x.md)      |
+| Data Table V2      | [Migrate](../../src/components/data-table/migrate-to-10.x.md)         |
```

Comment:
There's some changes in md files, was this some kind of formatting deal?

Reply:
This came in when I pulled ... there were some formatting changes, so not totally sure which are correct

Reply:
If there's no objections I'm going to leave this ... it would have me break the table styling